network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/SwitchImpl.java

```diff
@@ -83,7 +87,10 @@ public boolean isFictitious() {
 
     @Override
     public void setFictitious(boolean fictitious) {
+        boolean oldValue = resource.getAttributes().isFictitious();
         resource.getAttributes().setFictitious(fictitious);
+        String variantId = index.getNetwork().getVariantManager().getWorkingVariantId();
```

Comment:
in powsybl core, this notification doesn't use a variantId

Reply:
Corrected